src/tests/ftest/util/data_mover_utils.py

```diff
@@ -131,14 +131,32 @@ def set_datamover_params(self, src_pool=None, dst_pool=None, src_cont=None,
 class DataMover(DataMoverCommand):
     """Class defining an object of type DataMoverCommand."""
 
-    def __init__(self, hosts, timeout=30):
+    def __init__(self, hosts, tmp, timeout=30):
         """Create a datamover object."""
         super(DataMover, self).__init__(
             "/run/datamover/*", "dcp")
 
         # set params
         self.timeout = timeout
         self.hosts = hosts
+        self.tmp = tmp
```

Comment:
Maybe this is out of scope, but `tmp` doesn't look like a good variable name.

Reply:
I don't disagree, but `self.tmp` comes from deeper in the test framework, such as:
https://github.com/daos-stack/daos/blob/89a9fa8f0172a6ded9ba2dbdd41d955ae09d6a27/src/tests/ftest/util/apricot/apricot/test.py#L220
And then various utils use `self.tmp`.